### DISCUSSION TITLE
$ flyscrape template news cnn.js

### DISCUSSION THREAD

**Discussion by dynabler** [philippta/flyscrape#53] [Ideas]
Use templates when creating a new script.

Scenario: You want to create a news aggregator and will scrape x sites/feeds/RSS. It keeps your workflow neat if all your scripts are consistent, containing the same info (author, publish_date, source, etc.)

Inspiration: Hugo uses archetypes, and some command-line programs allow you to set aliases for a set of commands.

Implementation: In a folder where your flyscrape is installed. For example: `C:\flyscrape\bin\templates\news.js`. You then use `flyscrape template news cnn.js` (since template is new, no need for `new` again)

**Comment by philippta** [comment]
Thanks for sharing this idea. Although I like it, I'm not sure if we're quite there yet.

What I mean by that is, that I don't full understand yet how and in what ways people are using Flyscrape.

But if you're looking to have a number of ready made templates, there's nothing stopping you from having a personal folder somewhere for storing them. Then, whenever you like to use one of the scripts as a base, simply create a copy of it and work in that one.
